lib/user_feed_page.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:lab2/tweet_helper.dart';
+import 'package:lab2/tweet_item_model.dart';
+
+class UserFeedPage extends StatefulWidget {
+  @override
+  _UserFeedPageState createState() => _UserFeedPageState();
+}
+
+class _UserFeedPageState extends State<UserFeedPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white,
+        title: const Text(
+          "Home",
+          style: TextStyle(color: Colors.black),
+        ),
+        leading: const Icon(
+          Icons.account_circle,
+          color: Colors.grey,
+          size: 35.0,
+        ),
+      ),
+      body: ListView.builder(
+        itemCount: 4,
```

Comment:
What you're doing now is:
1. declaring hardcoded list of tweets (4 elements)
2. declaring a method that allows you to get single tweet by index
3. declaring hardcoded list widget with 4 elements, and manually mapping each list item to a tweet by index.

What you should do instead:
1. same, but ideally more than 4, to demonstrate scroll
2. declare a method that returns full collection instead 
3. your list widget should be built dynamically based on how many elements you have in that list, instead of hardcoding the length. You can either use for loop, or `list.map` method, and map every `Tweet` to a `Widget` that displays that tweet